src/services/checkov/executors/index.ts

```diff
@@ -1,2 +1,2 @@
-export * from './docker';
-export * from './pip3';
\ No newline at end of file
+export * from './DockerExecutor';
+export * from './Pip3Executor';
```

Comment:
@ChananM What's the change here exactly?

Reply:
@Saarett This file is actually unnecessary. I'll delete it.